controller/src/main/java/org/jboss/as/controller/remote/ModelControllerClientOperationHandler.java

```diff
@@ -78,6 +83,7 @@
  */
 public class ModelControllerClientOperationHandler implements ManagementRequestHandlerFactory {
 
+    private static final Set<String> PREPARED_RESPONSE_OPERATIONS = Stream.of(RELOAD, SHUTDOWN).collect(Collectors.toSet());
```

Comment:
I'm being a bit nitpicky, but I think `Arrays.asList(RELOAD, SHUTDOWN)` would be better here. The call stack to create this simple collection is huge with a stream.